
artificial_potential_field:

  attractive_constant: 0.1
  repulsive_constant: -0.5 # must be negative
  repulsive_threshold: 0.2
  speed_limit: 0.7
  error_radius: 0.05 # distance between goal and agent to determine if it has reached goal
  potential_field_timeout: 20